src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  let basePrice = 0;
+
+  for (let i = 0; i < days; i++) {
+    basePrice += oneDayCost;
+  }
```

Comment:
The loop to calculate `basePrice` is unnecessary. You can calculate the base price by multiplying `oneDayCost` by `days`. This will improve the performance by removing the loop.